.env.example

```diff
@@ -1,2 +1,3 @@
 CI=0
 DATABASE_URL="postgresql://daedalus-web-app-user:changeme@localhost:5432/daedalus-web-app"
+NUXT_PUBLIC_R_API_BASE=http://localhost:8001/
```

Comment:
This doesn't mean the API is public, it's just a way of automatically setting a config variable in nuxt.

Reply:
OK I managed to remove the 'PUBLIC' part :)